src/components/evidence-tag.tsx

```diff
@@ -8,18 +15,45 @@ import '../styles/components/evidence-tag.scss';
 
 const size = 12;
 
-const EvidenceTag = ({ label, title, className, iconComponent, children }) => {
+type EvidenceTagProps = {
+  /**
+   * Displayed on the tag
+   */
+  label: string;
+  /**
+   * Displayed on on mouseover
+   */
+  title: string;
```

Comment:
I would remove that as it's now included in `HTMLAttributes<HTMLButtonElement>>`, and also just let it flow with the rest of the `props` to the button instead of the span within